src/js/components/Pagination/__tests__/Pagination-test.js

```diff
@@ -0,0 +1,262 @@
+import React, { useState } from 'react';
+import 'jest-styled-components';
+import 'jest-axe/extend-expect';
+import 'regenerator-runtime/runtime';
+
+import { cleanup, render } from '@testing-library/react';
+// import { axe } from 'jest-axe';
+import { fireEvent } from '@testing-library/dom';
+
+import { Grommet } from '../../Grommet';
+import { Box } from '../../Box';
+import { Pagination } from '..';
+
+const NUM_ITEMS = 237;
+const STEP = 10;
+const data = [];
+for (let i = 0; i < 95; i += 1) {
+  data.push(`entry-${i}`);
+}
+
+const TestPagination = ({ step, ...rest }) => {
+  const [currentData] = useState(data.slice(0, step));
+
+  return (
+    <>
+      {currentData.map(datum => (
```

Comment:
Do we even need to have any data elements when testing Pagination?

Reply:
We don't have to. This was testing the case where the caller is controlling the page via state and confirming that the proper number of results displays. However, I'm realizing this is more a test of the caller's implementation vs something that the component is doing. I am going to remove this test.